Enable internal symbol aliases in order to not perform internal calls to
public interfaces through the plt within libxml.so.   

Index: libxml.h
--- libxml.h.orig
+++ libxml.h
@@ -120,7 +120,7 @@ int xmlInputReadCallbackNop(void *context, char *buffe
 #ifdef IN_LIBXML
 #ifdef __GNUC__
 #ifdef PIC
-#ifdef __linux__
+#if defined(__linux__) || defined(__OpenBSD__)
 #if (__GNUC__ == 3 && __GNUC_MINOR__ >= 3) || (__GNUC__ > 3)
 #include "elfgcchack.h"
 #endif
